### ISSUE TITLE
Feature Request - Sideshow Bob

### ISSUE THREAD

**Issue by Loxrie** [pattle/simpsons-in-css#6]
These are excellent, though I feel with the Lego Simpson's sets neglecting Sideshow Bob, perhaps he can be honoured here?

**Comment by pattle** [comment]
Yes Sideshow Bob would be a good addition. He is one of the hardest characters to do because his hair is very complex but I will attempt him soon
